suite-native/app/app.config.ts

```diff
@@ -81,6 +70,18 @@ export default ({ config }: ConfigContext): ExpoConfig => ({
     ios: {
         bundleIdentifier,
         icon: appIconIos,
+        infoPlist: {
```

Comment:
Is this used to define whole info.plist for builds and what is not here will not be in the final plist?

Reply:
https://github.com/trezor/trezor-suite/pull/10898/commits/2e44a6a9f018bf4679ec199a90de1d49ff780d8f